capabilities.json

```diff
@@ -249,10 +262,31 @@
                     "type": {
                         "text": true
                     }
+                },
+                "linkPositions": {
+                    "displayName": "Link positions",
+                    "displayNameKey": "Visual_LinkPositions",
```

Comment:
When you create a new **displayNameKey** entry, don't forget to add English string with the same key here:
https://github.com/microsoft/powerbi-visuals-sankey/blob/master/stringResources/en-US/resources.resjson#L30